### ISSUE TITLE
Whitespace should be enforced after await keyword

### ISSUE THREAD

**Issue by jordanbtucker** [palantir/tslint#4471]
### Rule Suggestion

**Is your rule for a general problem or is it specific to your development style?**
General problem.

**What does your suggested rule do?**
Checks for whitespace after `await`.

**List several examples where your rule could be used**
```ts
// Whitespace should be enforced after `await` in both of these examples.

const names = await(cache.get('.') || fs.promises.readdir('.'))

for await(const chunk of stream) {
  console.log(chunk)
}
```

**Comment by JoshuaKGoldberg** [comment]
Per #4534, closing this formatting rule request. If you _really_ need the rule, custom rules are always an option and can be maintained outside this repo!

**Comment by jordanbtucker** [comment]
Sounds good. I've since switched to using Prettier for formatting too.
